apps/site/src/lib/styles/globals.scss

```diff
@@ -2,3 +2,18 @@
 	background-image: url("~@/assets/background/anteater-head-tiling.gif");
 	background-size: 464px; // half size for 2x scaling
 }
+
+.accordion {
+	--bs-accordion-active-color: theme.$black;
+	--bs-accordion-bg: transparent;
+	--bs-accordion-active-bg: transparent;
+
+	--bs-accordion-border-color: transparent;
+	--bs-accordion-btn-focus-box-shadow: 0;
+	--bs-accordion-border-radius: 0;
+	--bs-accordion-btn-icon-width: 14px;
+
+	--bs-accordion-btn-icon: url("~@/assets/icons/accordion-btn.svg");
+	--bs-accordion-btn-active-icon: url("~@/assets/icons/accordion-btn.svg");
+	--bs-accordion-btn-icon-width: 25px;
+}
```

Comment:
Praise: nice work overriding the Bootstrap variables when there doesn't seem to be a SASS mixin.
Issue: we probably can't make all of these global since we also have an accordion for the FAQ on the home page.

Reply:
Good point. I kept the ones that are the same across both in this file and put the icon related variables in their separate SCSS files.